msal/src/Microsoft.Identity.Client/Platforms/netstandard1.3/WebUIFactory.cs

```diff
@@ -1,41 +0,0 @@
-﻿//----------------------------------------------------------------------
-//
-// Copyright (c) Microsoft Corporation.
-// All rights reserved.
-//
-// This code is licensed under the MIT License.
-//
-// Permission is hereby granted, free of charge, to any person obtaining a copy
-// of this software and associated documentation files(the "Software"), to deal
-// in the Software without restriction, including without limitation the rights
-// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
-// copies of the Software, and to permit persons to whom the Software is
-// furnished to do so, subject to the following conditions :
-//
-// The above copyright notice and this permission notice shall be included in
-// all copies or substantial portions of the Software.
-//
-// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
-// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-// THE SOFTWARE.
-//
-//------------------------------------------------------------------------------
-
-using System;
-using Microsoft.Identity.Core;
-using Microsoft.Identity.Core.UI;
-
-namespace Microsoft.Identity.Client.Internal.UI
-{
-    internal class WebUIFactory : IWebUIFactory
-    {
-        public IWebUI CreateAuthenticationDialog(CoreUIParent parent, RequestContext requestContext)
-        {
-            throw new NotImplementedException();
```

Comment:
The user experience for before you removed this was that if I call this library from a platform we do not fully support, for example WinPhone7 / NetCore or from a XForms app where I forget to add a reference to MSAL from XForms.UWP, I'd get a this exception. 

What is the user experience now?

I think it would be better to leave this as it is, and instead to throw a PlatformNotSupported exception stating: 

"Possible Cause: If you are using an XForms app, or generally a netstandard assembly, make sure you add a refernce to Microsoft.Identity.Client.dll from each platform assembly (e.g. UWP, Android, iOS), not just from the common netstandard assembly"

Reply:
yes, I agree. this would be much more actionable.

Reply:
sounds good. done. thank you.

Reply:
Makes sense. At least if we add that additional info.